app/frontend/components/admin/enrollments/Show.jsx

```diff
@@ -50,17 +159,7 @@ function AdminEnrollmentShow() {
         </thead>
         <tbody>
           {students.map((student, index) => (
-            <tr key={index}>
-              <td>{index + 1}</td>
-              <td>{student.first_name}</td>
-              <td>{student.last_name}</td>
-              <td>{student.email}</td>
-              <td>{student.phone}</td>
-              <td>
-                <Pencil color="blue" /> &emsp;
-                <Trash color="red" />
-              </td>
-            </tr>
+            populateRow(rowEdit, rowEditID, student, index)
```

Comment:
You'll have to explain this to me if it's doing the same thing as the original function. Seems like a great refactor.